test/tests/fvkernels/constraints/bounded_value.i

```diff
@@ -0,0 +1,73 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 1
+  nx = 4
+[]
+
+[Variables]
+  [v]
+    type = MooseVariableFVReal
+    # breaks the constraint
+    initial_condition = -1
+  []
+  [lambda]
+    type = MooseVariableScalar
+  []
+[]
+
+[FVKernels]
+  [time]
+    type = FVTimeKernel
+    variable = v
+  []
+  [diff]
+    type = FVDiffusion
+    variable = v
+    coeff = coeff
+  []
+  [average]
+    type = FVBoundedValueConstraint
+    variable = v
+    phi0 = 0
+    lambda = lambda
+    bound_type = 'HIGHER_THAN'
+  []
+[]
+
+[FVBCs]
+  [left]
+    type = FVDirichletBC
+    variable = v
+    boundary = left
+    value = 7
+  []
+  [right]
+    type = FVDirichletBC
+    variable = v
+    boundary = right
+    value = 0
+  []
+[]
+
+[Materials]
+  [diff]
+    type = ADGenericFunctorMaterial
+    prop_names = 'coeff'
+    prop_values = '1'
+  []
+[]
+
+[Executioner]
+  type = Transient
+  solve_type = 'PJFNK'
```

Comment:
why is this PJFNK? And no off-diagonal coupling in the preconditioner? If the LM is playing a role in this problem, I would expect convergence to not be very good without that coupling